frontend/dockerfile/dockerfile_test.go

```diff
@@ -1504,6 +1506,110 @@ COPY arch-$TARGETARCH whoami
 	}
 }
 
+func testExportLocalNoSplit(t *testing.T, sb integration.Sandbox) {
```

Comment:
nit: these should probably be part of `client_test.go`, since they test non-dockerfile specific behavior.